src/draw/lv_draw_mask_private.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file lv_draw_mask_private.h
+ *
+ */
+
+#ifndef LV_DRAW_MASK_PRIVATE_H
+#define LV_DRAW_MASK_PRIVATE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_draw_mask.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+struct lv_draw_mask_rect_dsc_t {
```

Comment:
make sure there are functions to set these fields because making it private without having those functions makes the structure unusable to a binding.

Reply:
It's internal only.